www/app/blog/2024-11-19-distributed-ddl.mdx

```diff
@@ -287,12 +287,12 @@ There is a special case for database failure, which is the case where the databa
 
 ### Recovering from coordinator failure
 
-If the coordinator itself fails, the first priority is to reconstruct its in-memory state so it can resume managing ongoing transactions and complete them. The coordinator first uses the latest state from the status tables in the metadata database, then checks the state of individual databases. For each transaction, a database can have it in one of three states: committed, prepared or non-existant. This leads to the following scenarios:
+If the coordinator itself fails, the first priority is to reconstruct its in-memory state so it can resume managing ongoing transactions and complete them. The coordinator first uses the latest state from the status tables in the metadata database, then checks the state of individual databases. For each transaction, a database can have it in one of three states: committed, prepared or nonexistent. This leads to the following scenarios:
```

Comment:
Not sure if you want the hyphen in here
```suggestion
If the coordinator itself fails, the first priority is to reconstruct its in-memory state so it can resume managing ongoing transactions and complete them. The coordinator first uses the latest state from the status tables in the metadata database, then checks the state of individual databases. For each transaction, a database can have it in one of three states: committed, prepared or non-existent. This leads to the following scenarios:
```